### ISSUE TITLE
@logux/client has a peer dependency conflict with nanostores

### ISSUE THREAD

**Issue by ZosoDerGoldene** [logux/logux#111]
npm i @logux/client gives the following output:
```
npm ERR! Found: nanostores@0.7.0
npm ERR! node_modules/nanostores
npm ERR!   peer nanostores@"^0.7.0" from @nanostores/preact@0.3.1
npm ERR!   node_modules/@nanostores/preact
npm ERR!     peerOptional @nanostores/preact@">=0.0.0" from @logux/client@0.18.2
npm ERR!     node_modules/@logux/client
npm ERR!       @logux/client@"*" from the root project
npm ERR!   peer nanostores@"^0.7.0" from @nanostores/react@0.4.0
npm ERR!   node_modules/@nanostores/react
npm ERR!     peerOptional @nanostores/react@">=0.0.0" from @logux/client@0.18.2
npm ERR!     node_modules/@logux/client
npm ERR!       @logux/client@"*" from the root project
npm ERR!   1 more (@nanostores/vue)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer nanostores@"^0.6.0" from @logux/client@0.18.2
npm ERR! node_modules/@logux/client
npm ERR!   @logux/client@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

```
so while everything else depends on nanostores 0.7.0, client seems to still depend on 0.6.0

**Comment by ai** [comment]
Thanks. I will update it soon.

**Comment by ai** [comment]
Fixed https://github.com/logux/client/commit/b6f17f05639ae32c8f59b86adc71377cb0c8238d

The fix was released in `0.18.3` version.
